CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
+## v7.1.0-beta
+
+### Features ✅
+
+ - Reduce startup time for Mesh node by only waiting for a block to be processed if Mesh isn't already sync'ed up to the latest block. ([#622](https://github.com/0xProject/0x-mesh/pull/622))
```

Comment:
nit: `sync'ed` => `synced`

Reply:
This release note was taken from `development` so I'm going to leave as is.